### ISSUE TITLE
Could not cast or convert from System.Boolean to C oiniumServ.Daemon.Responses.BlockTemplate

### ISSUE THREAD

**Issue by swkcc** [bonesoul/CoiniumServ#1083]
20:54:34 [Error] [Pool] [AlienChain] Pool initilization failed; System.Exception
: There was a problem deserializing the response from the coin wallet. ---> Newt
onsoft.Json.JsonSerializationException: Error converting value False to type 'Co
iniumServ.Daemon.Responses.BlockTemplate'. Path 'result', line 1, position 15. -
--> System.ArgumentException: Could not cast or convert from System.Boolean to C
oiniumServ.Daemon.Responses.BlockTemplate.
   在 Newtonsoft.Json.Utilities.ConvertUtils.EnsureTypeAssignable(Object value,
Type initialType, Type targetType)
   在 Newtonsoft.Json.Utilities.ConvertUtils.ConvertOrCast(Object initialValue,
CultureInfo culture, Type targetType)
   在 Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(Json
Reader reader, Object value, CultureInfo culture, JsonContract contract, Type ta
rgetType)
   --- 内部异常堆栈跟踪的结尾 ---
   在 Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(Json
Reader reader, Object value, CultureInfo culture, JsonContract contract, Type ta
rgetType)
   在 Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ResolveProperty
AndCreatorValues(JsonObjectContract contract, JsonProperty containerProperty, Js
onReader reader, Type objectType)
   在 Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObjectUsi
ngCreatorWithParameters(JsonReader reader, JsonObjectContract contract, JsonProp
erty containerProperty, ObjectConstructor`1 creator, String id)
   在 Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(Js
onReader reader, Type objectType, JsonContract contract, JsonProperty member, Js
onContainerContract containerContract, JsonProperty containerMember, Object exis
tingValue)
   在 Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(Jso
nReader reader, Type objectType, Boolean checkAdditionalContent)
   在 Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type
 objectType)
   在 Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, Jso
nSerializerSettings settings)
   在 Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSeriali
zerSettings settings)
   在 CoiniumServ.Daemon.DaemonBase.GetRpcResponse[T](HttpWebRequest httpWebRequ
est) 位置 C:\pool\src\CoiniumServ\Daemon\DaemonBase.cs:行号 188
   --- 内部异常堆栈跟踪的结尾 ---
   在 CoiniumServ.Daemon.DaemonBase.GetRpcResponse[T](HttpWebRequest httpWebRequ
est) 位置 C:\pool\src\CoiniumServ\Daemon\DaemonBase.cs:行号 193
   在 CoiniumServ.Daemon.DaemonBase.MakeRpcRequest[T](DaemonRequest walletReques
t) 位置 C:\pool\src\CoiniumServ\Daemon\DaemonBase.cs:行号 102
   在 CoiniumServ.Daemon.DaemonBase.MakeRequest[T](String method, Object[] param
eters) 位置 C:\pool\src\CoiniumServ\Daemon\DaemonBase.cs:行号 88
   在 CoiniumServ.Daemon.DaemonClient.GetBlockTemplate(Boolean modeRequired) 位
置 C:\pool\src\CoiniumServ\Daemon\DaemonClient.cs:行号 271
   在 CoiniumServ.Jobs.Manager.JobManager.GetNewJob() 位置 C:\pool\src\CoiniumSe
rv\Jobs\Manager\JobManager.cs:行号 163                                        䶶ﬀ弯ŘŘ䶘픀弯ŘŘ

**Comment by Casper0513** [comment]
is this the only issue you get??? im running Ubuntu 19.04 and all kinds of issues especially for the wallets

**Comment by swkcc** [comment]
Yes, that's all. I think it's the wallet that returns the data and parses the problem.

**Comment by Casper0513** [comment]
have you got a fix for the issue????

**Comment by swkcc** [comment]
> have you got a fix for the issue????

no
